public/app/core/components/sidemenu/BottomNavLinks.tsx

```diff
@@ -50,8 +50,8 @@ export default class BottomNavLinks extends PureComponent<Props, State> {
       <div className="sidemenu-item dropdown dropup">
         <Link href={link.url} className="sidemenu-link" target={link.target}>
           <span className="icon-circle sidemenu-icon">
-            {link.icon && <Icon name={link.icon as IconName} size="xl" />}
-            {link.img && <img src={link.img} />}
+            {link.icon && <Icon name={link.icon as IconName} size="xl" title="Help icon" />}
```

Comment:
Probably a silly question but is this always a help icon?

Reply:
Yeah, I think so. Unless you start adding bottom nav links, it's going to be the same